src/posts/en/configuring-babel-6-for-node-js.md

```diff
@@ -0,0 +1,164 @@
+<!--
+layout: post
+title: Configuring Babel 6 for Node.js
+date: 2015-12-31T01:23:42.811Z
+comments: true
+published: true
+keywords: JavaScript, Node.js, ES6, babel, transpiler
+description: Tutorial on configuring babel for node.js to get up and running quickly
+categories: ES6, babel, node.js
+authorName: Hannan Ali
+authorLink: https://abdulhannanali.github.io
+authorPicture: //s.gravatar.com/avatar/89e5f7614cb88cd573359a953a09aa6e?s=80
+-->
+Hi! If you are like me, you are tired of writing the same old ES5 JS Code in your node applications.
+
+If yes, you can use the newer features of JavaScript ES6 and ES7 standards in your node applications today. ES6 and ES7 make the JavaScript development a cool breeze. But hey, not every ES6 feature is supported in our beloved [Node.js](https://nodejs.org).
+
+This is where [BabelJS](https://babeljs.io) comes to the rescue. BabelJS is a transpiler for JavaScript which transpiles your ES6 and ES7 code into ES5 and even ES3 code. In simple words it converts it into JavaScript that node.js can run and make you really happy.
+<!--more-->
+
+**Little Notice:** If you just want ES6 features and don't want to babelify stuff. You can use `--harmony flag` before running your node application. In order to access more harmony flags for staging and experimental features run this command `node --v8-options | grep harmony` . But there aren't even all the features present in Node. So you may continue if you can access ES6 but also need to access **ES7** features.
+
+### Some assumptions made
+There are some assumptions I am making about you! YES YOU!
+- You know your way around [Node.JS](https://nodejs.org)
+- You can install packages using [npmjs.com](http://npmjs.com)
+- You've both of 'em Node.js and NPM already installed.
+- You are okay with using some CLI sometimes.
+- It's good to know some ES6 or ES7 beforehand but not required.
+
+
+### Following along the code
+Type of person who follows code instead of just reading? Code is available here in this [repo](https://github.com/abdulhannanali/babel-configuration-tutorial)
+
+### Installing and getting started with babeljs
+There's not a single way you will be able to setup Babel. Here we will be discussing enough to get up and running with it.
```

Comment:
"There's not a single way you will be able to setup Babel."
I think this statement could be misinterpreted as "There is no way you will be able to setup Babel.". Perhaps change "There's not a single way" to "There is more than one way".